### PR TITLE
Fix 4671

### PR DESCRIPTION
**Pull request by jingshanglu**
ref #4671 .

Changes proposed in this pull request:
- delete afterColumName from ColumnAfterPositionSegment
- fix testCase about position column

### CODE REVIEW

**Comment by jingshanglu** [comment]
@ tristaZero

File: shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/test/java/org/apache/shardingsphere/sql/parser/integrate/asserts/segment/definition/ColumnPositionAssert.java [hunk 1]
```diff
@@ -47,15 +47,19 @@ public static void assertIs(final SQLCaseAssertContext assertContext, final Colu
         if (null != actual.getColumnName()) {
             columnname = actual.getColumnName().getQualifiedName();
         }
```
Comment (tristaZero) [inline shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/test/java/org/apache/shardingsphere/sql/parser/integrate/asserts/segment/definition/ColumnPositionAssert.java]: It is better to extract a function.

File: shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/test/java/org/apache/shardingsphere/sql/parser/integrate/asserts/segment/definition/ColumnPositionAssert.java [hunk 2]
```diff
@@ -47,15 +47,19 @@ public static void assertIs(final SQLCaseAssertContext assertContext, final Colu
...
-        assertThat(assertContext.getText("Column change position name assertion error: "), columnname, is(expected.getColumn().getName()));
+        String expectColumnName = null;
+        if (null != expected.getColumn()) {
+            expectColumnName = expected.getColumn().getName();
+        }
```
Comment (tristaZero) [inline shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/test/java/org/apache/shardingsphere/sql/parser/integrate/asserts/segment/definition/ColumnPositionAssert.java]: This situation is similar to the above one.

**Comment by tristaZero** [comment]
Hi@jingshanglu, Your PR is great, but could you look at my comment which can make this PR perfect, I think. Also, welcome your input. :)

**Comment by jingshanglu** [comment]
> Hi@jingshanglu, Your PR is great, but could you look at my comment which can make this PR perfect, I think. Also, welcome your input. :)

These codes are only used in this place, there is no need to draw a method

**Comment by tristaZero** [comment]
@jingshanglu Ok, it is not a big deal, thanks for your feedback.

**Pull request merged**